build_tools/unittest.sh

```diff
@@ -0,0 +1,18 @@
+set -ex
+
+# https://stackoverflow.com/a/246128
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )  # p4mlir/build_tools
+P4MLIR_REPO_DIR=$( cd "$SCRIPT_DIR"/.. &> /dev/null && pwd )
+
+P4C_REPO_DIR=$P4MLIR_REPO_DIR/third_party/p4c
+P4C_BUILD_DIR=$P4C_REPO_DIR/build
+P4C_EXT_DIR=$P4C_REPO_DIR/extensions
+
+P4C_P4MLIR_EXT_DIR=$P4C_EXT_DIR/p4mlir
+
+cd "$P4C_BUILD_DIR"
+# Run some tests
+ninja check-p4mlir
```

Comment:
For the purpose of running unit test, you can simply go to the build directory and run `ninja check-p4mlir`. There is no need for a separate script.

This is also written in the project README already.

Reply:
Just to test the build system to see why it hangs, not trying to merge the file, just separate the file. Local build and test are fine.

Reply:
Cool. Thanks for the explanation. That's fine!